src/core/useEvent.ts

```diff
@@ -22,9 +22,13 @@ export function useEvent<D>(
   useEffect(() => {
     if (channel === undefined) {
       return;
+    } else if (Array.isArray(channel)) {
+      channel.forEach((cn) => cn.bind(eventName, callback));
```

Comment:
Is the idea to send the same message to multiple channels? What if we want to send an event to just one - is that passed in to the `useEvent` hook alone?

Reply:
Yes, to listen for an event on multiple channels. Yes, they can pass in a singular channel or an array of channels so it's flexible.